chronograf.go

```diff
@@ -644,6 +644,10 @@ type Axis struct {
 	Bounds       []string `json:"bounds"` // bounds are an arbitrary list of client-defined strings that specify the viewport for a cell
 	LegacyBounds [2]int64 `json:"-"`      // legacy bounds are for testing a migration from an earlier version of axis
 	Label        string   `json:"label"`  // label is a description of this Axis
+	Prefix       string   `json:"prefix"` // specifies a prefix for axis values
```

Comment:
should use the name of the member as first word of comment (e.g. // Prefix represents a label prefix for formatting axis values.